app/models/skill_set.rb

```diff
@@ -9,4 +9,10 @@ def attributes(display_price = 1)
        "health" => health
      }
   end
+
+  def non_zero_attributes
+    attributes.select do |k,v|
```

Comment:
Unused block argument - `k`. If it's necessary, use `_` or `_k` as an argument name to indicate that it won't be used.<br>Space missing after comma.